packages/web/pages/stake/index.tsx

```diff
@@ -265,9 +265,9 @@ export const Staking: React.FC = observer(() => {
   const showStakeLearnMore = !isWalletConnected || isNewUser;
 
   return (
-    <main className="relative flex h-screen items-center justify-center">
-      <div className="flex w-full justify-center space-x-5">
-        <div>
+    <main className="flex h-full items-center justify-center px-6 py-8 lg:relative lg:items-start">
+      <div className="grid max-w-[1178px] grid-cols-2 grid-cols-[1fr,2fr] gap-4 lg:max-w-full lg:max-w-[478px] lg:grid-cols-1 lg:gap-y-4">
+        <div className="flex flex-col gap-4">
```

Comment:
If possible, could we prefer rem value over px? This would help us scale the user's view with their preferred font size.

If Figma is using pixel values, I find this tool helpful for finding the associated rem value for the standard default font size

https://nekocalc.com/px-to-rem-converter